MbDotNet.Tests/Acceptance/DocumentationTests.cs

```diff
@@ -449,6 +449,36 @@ public async Task JsonExample()
 
             await _client.SubmitAsync(imposter);
         }
+
+        /// <summary>
+        /// This test shows how to setup the imposter in the inject predicate example
+        /// at http://www.mbtest.org/docs/api/predicates.
+        /// </summary>
+        [TestMethod]
+        public async Task HttpInjectPredicateExample()
+        {
+            var imposter = _client.CreateHttpImposter(4546, "HttpInjectPredicateExample");
+
+            const string javaScriptFunction = "function(config) { return true; }";
+            imposter.AddStub().OnJavaScriptFunction(javaScriptFunction).ReturnsBody(HttpStatusCode.OK, "matches");
+
+            await _client.SubmitAsync(imposter);
+        }
+
+        /// <summary>
+        /// This test shows how to setup the imposter in the inject predicate example
+        /// at http://www.mbtest.org/docs/api/predicates.
+        /// </summary>
+        [TestMethod]
+        public async Task TcpInjectPredicateExample()
+        {
+            var imposter = _client.CreateTcpImposter(4547, "TcpInjectPredicateExample");
+
+            const string javaScriptFunction = "function(config) { return true; }";
+            imposter.AddStub().OnJavaScriptFunction(javaScriptFunction).ReturnsData("matches");
+
+            await _client.SubmitAsync(imposter);
+        }
```

Comment:
Generally I use this file to include tests that match what is in the documentation for the feature. In this case, it would be the predicate injection example shown here: http://www.mbtest.org/docs/api/injection

That way if someone is looking at the mountebank docs and is wondering how they would create the same imposter in MbDotNet, they can look in this file.

Could you please replace these tests with a single test that sets up this imposter (from the docs):
```
{
  "port": 4545,
  "protocol": "http",
  "stubs": [
    {
      "responses": [{ "is": { "statusCode": 400 } }],
      "predicates": [{
        "inject": "function (config) {\n\n    function hasXMLProlog () {\n        return config.request.body.indexOf('<?xml') === 0;\n    }\n\n    if (config.request.headers['Content-Type'] === 'application/xml') {\n        return !hasXMLProlog();\n    }\n    else {\n        return hasXMLProlog();\n    }\n}"
      }]
    }
  ]
}
```